llvm/test/Bindings/OCaml/passbuilder.ml

```diff
@@ -0,0 +1,50 @@
+(* RUN: rm -rf %t && mkdir -p %t && cp %s %t/passbuilder.ml
+ * RUN: %ocamlc -g -w +A -package llvm.passbuilder -package llvm.all_backends -linkpkg %t/passbuilder.ml -o %t/executable
+ * RUN: %t/executable
+ * RUN: %ocamlopt -g -w +A -package llvm.passbuilder -package llvm.all_backends -linkpkg %t/passbuilder.ml -o %t/executable
+ * RUN: %t/executable
+ * XFAIL: vg_leak
+ *)
+
+let () = Llvm_all_backends.initialize ()
+
+(*===-- Fixture -----------------------------------------------------------===*)
+
+let context = Llvm.global_context ()
+
+let m = Llvm.create_module context "mymodule"
```

Comment:
It would perhaps be a more effective test if this loaded something like llvm/test/Other/new-pass-manager.ll rather than just using an empty module.